Add verification cases ver-1i (#182)
idaholab/TMAP8: doc/content/verification_and_validation/ver-1ic.md

# ver-1ic

# Species Equilibration Model in Surfdep Conditions with Low Barrier Energy

## General Case Description

> <!-- All necessary equations -->

Reviewer: ```suggestion
```

The problem considers the reaction between two isotopic species, $A_2$ and $B_2$, on a surface in surfdep condition. The reaction is described as

\begin{equation}
\label{eq:reaction}
\frac{1}{2} A_2 + \frac{1}{2} B_2 \rightleftharpoons AB.
\end{equation}

and the partial pressure of $A_2$, $B_2$, and $AB$ in equilibrium of the reaction is defined by

\begin{equation}
\label{eq:reaction_equilibrium_pressure}
\frac{P_{AB}}{\sqrt{P_{A_2}} \sqrt{P_{B_2}}} = K_{eq},
\end{equation}

where $P_i$ is the partial pressure of corresponding gas $i$, $K_{eq}$ is the equilibrium constant, and $K_{eq} = 2$ in this isotope reaction. Therefore, the partial pressure of $AB$ in equilibrium is a constant value depends on initial partial pressure of $A_2$ and $B_2$:

\begin{equation}
\label{eq:p_AB_equilibrium}
P_{AB}^{eq} = \frac{2 P_{A_2}^0 P_{B_2}^0}{P_{A_2}^0 + P_{B_2}^0}.
\end{equation}

> Under surfdep condition, the conversion of A2 and B2 molecules to AB molecules requires several steps. First, homonuclear molecules in the gas must get to the surface. Next, they must dissociate. Then the individual surface atoms must migrate to sites where they encounter their conjugates. Here we assume there is a probability of unity of their combination once they find each other. Finally, the AB molecule must leave the surface and return to the gas. We solve

Reviewer: ```suggestion
Under surfdep condition, the conversion of A$_2$ and B$_2$ molecules to AB molecules requires several steps. First, homonuclear molecules in the gas must get to the surface. Next, they must dissociate. Then the individual surface atoms must migrate to sites where they encounter their conjugates. Here we assume there is a probability of unity of their combination once they find each other. Finally, the AB molecule must leave the surface and return to the gas. We solve
```

\begin{equation}
\label{eq:k_d_equation}
\hat{K_d} = \frac{1}{\sqrt{2 \pi M k T}} \exp \left( - \frac{E_x}{k T} \right),
\end{equation}

\begin{equation}
\label{eq:k_r_equation}
\hat{K_r} = \nu_0 \exp \left( \frac{E_c - E_x}{k T} \right),
\end{equation}

and

\begin{equation}
\label{eq:k_b_equation}
\hat{K_b} = \nu_0 \exp \left( - \frac{E_b}{k T} \right),
\end{equation}

where $M$ is the mass of species molecules, $\nu_0$ is the Debye frequency, $E_x$ is the adsorption barrier energy, $E_c$ is the surface binding energy, and $E_b$ is the dissociation activation energy. The production of $C_A$ and $C_B$ in equilibration is given by

\begin{equation}
\label{eq:equal_c_a_c_b}
C_A C_B = \frac{\hat{K_b} \hat{K_d}}{2 D_s \lambda \hat{K_r}} P_{AB}^{eq}.
\end{equation}

> <!-- Detail parameters -->

Reviewer: ```suggestion
```

Please remove all these comments.

\end{equation}

where $t$ is the time, $S$ is the surface area, $k_b$ is the Boltzmann’s constant, $T$ is the temperature, $V$ is the volume in the enclosure. The production of A$_2$ and B$_2$ in equilibration conditions is given by

\begin{equation}
\label{eq:equal_c_a_c_b}
C_A C_B = \frac{\hat{K_b} \hat{K_d}}{2 D_s \lambda \hat{K_r}} P_{AB}^{eq}.
\end{equation}

This case uses equal starting pressures of $1 \times 10^{4}$ Pa of A$_2$ and B$_2$ and no AB. $E_x$, $E_c$, and $E_b$ are specified to be 0.05, -0.01, and 0.0 eV, respectively, $\nu_0$ is $8.4 \times 10^{12}$ m/s, the temperature is 1000 K, the surface area for reaction is 0.05 m $\times$ 0.05 m square, and the enclosure volume is 1 m$^3$.

## Analytical solution

[!cite](ambrosek2008verification) provides the analytical equation for the partial pressure of AB as

\begin{equation}
\label{eq:analytical_solution}
P_{AB}  = \frac{2 P_{A_2}^0 P_{B_2}^0}{P_{A_2}^0 + P_{B_2}^0} \left(1 - \exp \left(- \frac{t}{\tau}\right)\right),
\end{equation}

where $\tau$ is defined as

\begin{equation}

> \label{eq:tao}

Reviewer: ```suggestion
\label{eq:tau}
```
